server/utils.go

```diff
@@ -220,3 +221,19 @@ func getUserIDsFromSessions(sessions map[string]*public.CallSession) []string {
 	}
 	return userIDs
 }
+
+func (p *Plugin) getTranslationFunc(locale string) i18n.TranslateFunc {
+	if locale != "" {
+		return i18n.GetUserTranslations(locale)
+	}
+
+	locale = "en"
+	cfg := p.API.GetConfig()
+	if cfg == nil {
+		p.LogError("failed to get configuration")
+	} else if cfg.LocalizationSettings.DefaultClientLocale != nil {
+		locale = *cfg.LocalizationSettings.DefaultClientLocale
+	}
```

Comment:
We are currently defaulting to this to determine the locale for messages in call posts, given that they are channel-wide.

Reply:
Makes sense... I hope they set it. :)